ASP.NET API/WAVC_WebApi/Hubs/Interfaces/IFriendRequestClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+
+namespace WAVC_WebApi.Hubs.Interfaces
+{
+    public interface IFriendRequestClient
+    {
+        Task FriendDeleted(string id);
+
+        Task RecieveFriendRequest(string id, string name, string surname);
```

Comment:
Just id would be enough.

Reply:
We'd have to assume that client knows that users name and surname or client would have to make a seperate call to backend just to ask for this data. This way doesn't require user to make multiple calls.

Reply:
The incoherent function name made me confused. It's used for sending FR. Nonetheless I wound encapsulate all these variables in a separate model.. I bet there's something like that already in this project.